### PR TITLE
KJH

### PR DESCRIPTION
**Pull request by enfycius**

### CODE REVIEW

**Review by Hyunul — APPROVED** [review]

File: KJH/KJH.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+content
+content
+content
```
Comment (Hyunul) [inline KJH/KJH.txt:3]: 123123

Comment (sondonghan) [inline KJH/KJH.txt:3]: .

**Review by sondonghan — DISMISSED** [review]

**Review by Hyunul — APPROVED** [review]

**Review by Hyunul — COMMENTED** [review]
1231

**Review by Hyunul — DISMISSED** [review]

**Review by sondonghan — APPROVED** [review]

**Review by Hyunul — APPROVED** [review]

File: KJH/test.ipynb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (enfycius) [inline KJH/test.ipynb:1]: <p>SO SO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DoWuJo/test/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (enfycius) [inline KJH/test.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    print("test")
</pre><p>SO SO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DoWuJo/test/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Pull request closed without merging**